metron-interface/metron-alerts/src/app/pcap/pcap-filters/pcap-filters.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Some of these .ts files don't have license headers, how is this passing the rat check?  Do we have a rat exclusion that is too broad?

Reply:
Thanks, we added license headers.